src/NServiceBus.Core.Tests/Serializers/XML/Issue_2796.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace NServiceBus.Core.Tests.Serializers.XML
+{
+    using System;
+    using System.IO;
+    using NServiceBus.Serializers.XML.Test;
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class Issue_2796
```

Comment:
> Is there a value in storing the issue number as metadata somehow?

In some cases, perhaps, but I think usually there's no need. In most cases, the test should be self describing and contain all necessary context. In those cases where an outside reference really is justified, I tend to write a comment with a link, but perhaps there is semantically stronger technique.

I used to be a big proponent of issue numbers in commits but I changed my mind some time back because in some projects, e.g. scriptcs, we found that this litters the issue with too many granular links. I tend to favour just linking PR's to issues so that all the commits are linked just once.